Prog/SF_2010_14/Acs_sf_2010_14_dc.sas

```diff
@@ -34,11 +34,11 @@
   table_list = 
   	B00001 B00002 B01002 B01003 
 	B01001 B01001B B01001C B01001D B01001E B01001F B01001G B01001H B01001I
-	B02001 B03002 B05002 B06002 B06007 B06009 B07012 B09001
+	B02001 B03001 B03002 B05002 B05003I B06002 B06007 B06009 B07012 B09001
     B06004 B06004B B06004C B06004D B06004E B06004F B06004G B06004H B06004I
```

Comment:
There is no B06004 table. B06004 should be deleted from this list. (B06004B B06004C etc. are fine.)
The revisions= parameter value should be changed to %str(Add tables for State of Latinos project.)